source/reference/disaster-recovery.txt

```diff
@@ -141,3 +153,22 @@ To enable writes, update :dbcommand:`setUserWriteBlockMode`:
        }
    )
 
+.. _c2c-dr-capped-collections:
+
+Capped Collections
```

Comment:
Hey @ajhuh-mdb I don't see this change as part of the original [PR](https://github.com/mongodb/docs-cluster-to-cluster-sync/pull/340/files). I'm guessing this was added to resolve a build error, but just want to confirm before I merge that it wasn't added by mistake. Thanks!

Reply:
Yep, that's correct! There was a build error that required this section to be added. 👍